azure-pipelines.yml

```diff
@@ -2,7 +2,9 @@
 
 # The code branch that will trigger a build
 trigger:
-- develop
+- feature/*
+- bugfix/*
+- refactor/*
 
```

Comment:
Will this pipeline still deploy the code? I can see it is outputting the artefact.

Reply:
Yes, until i have updated the pipeline, but as no one is working on it right now it shouldn't be an issue.